modules/lvms-restoring-volume-snapshots.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-restoring-volume-snapshots_{context}"]
+= Restoring volume snapshots
+
+To restore a volume snapshot, you must create a `PersistentVolumeClaim` custom resource (CR) with the `dataSource.name` field set to the name of the volume snapshot. 
+
+When you restore a volume snapshot, {lvms} creates a new persistent volume claim (PVC). The restored PVC is independent of the volume snapshot and the source PVC.
```

Comment:
LVMS does not create another PVC. The restored PVC is the one that the user just created.

Reply:
Removed the following line:
"When you restore a volume snapshot, {lvms} creates a new persistent volume claim (PVC)."